src/modules/blockchainApplication/explore/components/BlockchainApplicationDetails/BlockchainApplicationDetails.js

```diff
@@ -82,49 +92,37 @@ const BlockchainApplicationDetails = ({ history, location, application }) => {
             </TertiaryButton>
           </div>
           <div className={styles.addressRow}>
-            <a
-              className={`${styles.appLink}`}
-              target="_blank"
-                // eslint-disable-next-line
-                // TODO: this is just a placeholder link pending when its part of the response payload from service
-              href={serviceUrl}
-            >
+            <a className={`${styles.appLink}`} target="_blank" href={projectPage}>
               <Icon name="chainLinkIcon" className={styles.hwWalletIcon} />
-              {t(serviceUrl)}
+              {t(projectPage)}
             </a>
           </div>
           <div className={styles.balanceRow}>
             <span>{t('Deposited:')}</span>
             {/* TODO: this is a placeholder value pending when its part of service response */}
             <span>
-              <TokenAmount val={deposit} />
-              {' '}
-              LSK
+              <TokenAmount val={deposit} /> LSK
```

Comment:
LSK shouldn't be hardcode

Reply:
@soroushm There's a todo added there pending when the data from service is available, then it won't be hardcoded anymore